components/messages/HowTo.module.css

```diff
@@ -0,0 +1,27 @@
+.howtocontainer {
```

Comment:
Use camel case to make the class easier to read
```suggestion
.howToContainer {
```